api/app/controllers/plans_controller.rb

```diff
@@ -130,6 +167,6 @@ def set_user_plan
   end
 
   def authorized
-    @current_user_id == Integer(params[:user_id])
+    @current_user_id == Integer(params[:user_id]) || @user.is_advisor
```

Comment:
An advisor can pretty much do any action so I think we can just put this here instead of on every instance